### ISSUE TITLE
Create "How to get started with Ona" video 

### ISSUE THREAD

**Issue by COpenda** [onaio/help#219]

**Comment by rowo** [comment]
@COpenda I don't think you can fit the contents of a long, technical guide into a 60 second video. It would be helpful to get your marketing viewpoint on this. So, can you start by putting together content on what you think would realistically be communicated in a 60 second video into a get started guide that you would send to people instead of the long guide?

**Comment by COpenda** [comment]
@rowo getting back to this, I was thinking of having a short demo of how to upload a form, create projects, viewing data and exporting, not the entire guide. Based on the survey stats, users are having issues with getting started and it seems textual content hasn't been very helpful. Realistically, it may be longer than 60 seconds if we want to make it very detailed e.g https://www.youtube.com/watch?v=2LU6UYD4hpI. But if it's something like this, it can fit in a 60 second video https://www.youtube.com/watch?v=UeLyU0JeQ7k.

**Comment by rowo** [comment]
@COpenda My previous point had more to do with there not needing to be a time limit since anything you do will not be valuable in that timeframe. That Slack video is not even relevant since it only covers signing up. Look at the official Slack intro.. it's just under 3 min. That seems reasonable. You should do a few tests to/for yourself to figure out what's good for us and then propose it. The only tricky part is the authoring bit. https://stage-help.ona.io/knowledge-base/guide-my-first-project/ has the basics.

**Comment by rowo** [comment]
My guess is the main part people are having trouble with isn't signing up for Ona or creating a project or looking at data, but authoring the form (including uploading it and finding out it has errors).

If someone figured out how to author a form and upload it, exporting the data and viewing it is a piece of cake after seeing where to find them.
